src/components/Calculator.js

```diff
@@ -1,31 +1,52 @@
 import React from 'react';
 import '../components/App.css';
+import calculate from '../logic/calculate';
```

Comment:
The Calculator component imports the calculate.js file from the logic directory ✔